extensions-core/stats/src/main/java/io/druid/query/aggregation/teststats/ZtestPostAggregator.java

```diff
@@ -46,79 +48,92 @@
    http://facweb.cs.depaul.edu/sjost/csc423/documents/test-descriptions/indep-z.pdf
 */
 @JsonTypeName("zscore2sample")
-public class ZtestPostAggregator implements PostAggregator {
+public class ZtestPostAggregator implements PostAggregator
+{
   private final String name;
-  private final List<PostAggregator> fields;
+  private final PostAggregator successCount1;
+  private final PostAggregator sample1Size;
+  private final PostAggregator successCount2;
+  private final PostAggregator sample2Size;
```

Comment:
Also please add a json serde test.

Reply:
These two postAggregators implement PostAggregator Interface. Get() is only needed for Aggregator Interface. I failed ti see any postAggregator implementing get() method.

Reply:
Not sure why json serde is needed in this case? These two postAggregators didn’t persist into Druid by per se. And there is no custom Serde either.
One can think of them of glorified arithmetic post aggregators to calculate some test statistics.

Reply:
These two postAggregators implement PostAggregator Interface. Get() is only needed for Aggregator Interface. I failed to see any postAggregator implementing get() method.

Reply:
Not sure why json serde is needed in this case? These two postAggregators didn’t persist into Druid by per se. And there is no custom Serde either.
One can think of them of glorified arithmetic post aggregators to calculate some test statistics.

Reply:
Looks that my comments were confused. Sorry for lack of details. I meant, the series of methods of getSuccessCount1(), getSample1Size(), getSuccessCount2(), and getSample2Size() should be added and annotated with ```@JsonProperty``` because they are needed for JSON serialization. Unit tests are needed to avoid such mistakes.

Reply:
Thanks for clarifying this!